### ISSUE TITLE
Missing HitArmor Entry

### ISSUE THREAD

**Issue by CptFunkymonk** [Tinter/RusWepPack#35]
Noticed this for a couple of shotguns, specifically the TOZ in this case.

http://imgur.com/sh6vyxk

Happens the second you shoot.

**Comment by Tinter** [comment]
Yeah, noticed that, had trouble getting it consistently though.
Toadie has a fix for it I'm told, gonna ask.
For the record, looks like it's looking for a config entry in the base config file. No idea what would causes that personally.

**Comment by Tinter** [comment]
Confirm that this is still an issue? Haven't noticed it myself in a while.

**Comment by CptFunkymonk** [comment]
Yes this is still an issue, I didn't happen the first shot, it happened the
second.

On Tue, Jun 7, 2016 at 9:11 AM, Tinter notifications@github.com wrote:

> Confirm that this is still an issue? Haven't noticed it myself in a while.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/Tinter/RusWepPack/issues/35#issuecomment-224292579,
> or mute the thread
> https://github.com/notifications/unsubscribe/ARbpWBanTTdvsMVLcfJQmAG800MF3TLAks5qJXwcgaJpZM4IBWYr
> .

## 

David Gonzalez-Gimenez
Iowa State - Industrial Technology
800 Pinon Dr, Unit 106
Ames, IA 50013
(630) 456-8934
